[2023-01-28 14:03:04] [WARNING]: 恢复会话失败: exchange_emp failed: 15 , 尝试使用正常流程登录. 
[2023-01-28 14:03:29] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2023-01-28 14:03:31] [WARNING]: 警告: 心跳功能已关闭，若非预期，请检查配置文件。 
[2023-01-28 14:03:31] [WARNING]: HTTP 服务器使用了过时的配置格式，请更新配置文件！ 
[2023-01-28 14:19:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 14:19:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 14:20:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 14:20:04] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674886791,"self_id":3351593463,"sub_type":"normal","font":0,"message_type":"group","user_id":2919465681,"message_id":2112577541,"anonymous":null,"group_id":542926457,"message":"我原来就打开了，结果发不了群聊","message_seq":18000,"raw_message":"我原来就打开了，结果发不了群聊","sender":{"age":0,"area":"","card":"","level":"","nickname":"万能的小黑","role":"member","sex":"unknown","title":"","user_id":2919465681}}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 14:32:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 14:32:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 14:32:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 14:32:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 14:32:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 14:32:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 14:32:18] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674887525,"self_id":3351593463,"sub_type":"normal","sender":{"age":0,"area":"","card":"","level":"","nickname":"洛瑜","role":"member","sex":"unknown","title":"","user_id":672545360},"raw_message":"[CQ:image,file=4a8378cc2692b86d8051a337080aeeea.image,subType=0,url=https://gchat.qpic.cn/gchatpic_new/672545360/542926457-2207793964-4A8378CC2692B86D8051A337080AEEEA/0?term=3\u0026amp;is_origin=0]","user_id":672545360,"anonymous":null,"font":0,"group_id":542926457,"message":"[CQ:image,file=4a8378cc2692b86d8051a337080aeeea.image,subType=0,url=https://gchat.qpic.cn/gchatpic_new/672545360/542926457-2207793964-4A8378CC2692B86D8051A337080AEEEA/0?term=3\u0026amp;is_origin=0]","message_type":"group","message_seq":18001,"message_id":183259283}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 14:32:20] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674887526,"self_id":3351593463,"sub_type":"normal","anonymous":null,"group_id":542926457,"raw_message":"这图咋样","user_id":672545360,"font":0,"message":"这图咋样","message_type":"group","message_seq":18002,"sender":{"age":0,"area":"","card":"","level":"","nickname":"洛瑜","role":"member","sex":"unknown","title":"","user_id":672545360},"message_id":-1813708503}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 14:32:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 14:32:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 14:32:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 14:32:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 14:32:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 14:32:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 14:32:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 14:32:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 14:32:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 14:32:34] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674887541,"self_id":3351593463,"sub_type":"normal","message_type":"group","user_id":3107856775,"message_id":-454938177,"message_seq":18003,"raw_message":"还行","sender":{"age":0,"area":"","card":"","level":"","nickname":"jerry","role":"admin","sex":"unknown","title":"易语言py","user_id":3107856775},"anonymous":null,"font":0,"group_id":542926457,"message":"还行"}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 14:32:35] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674887542,"self_id":3351593463,"sub_type":"normal","message_id":2055644188,"anonymous":null,"message_type":"group","message_seq":18004,"raw_message":"可以哦","sender":{"age":0,"area":"","card":"","level":"","nickname":"Starry Sea","role":"owner","sex":"unknown","title":"","user_id":2491267482},"user_id":2491267482,"font":0,"group_id":542926457,"message":"可以哦"}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 14:32:36] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674887543,"self_id":3351593463,"sub_type":"normal","anonymous":null,"message":"[CQ:face,id=277]","message_seq":18005,"raw_message":"[CQ:face,id=277]","sender":{"age":0,"area":"","card":"","level":"","nickname":"jerry","role":"admin","sex":"unknown","title":"易语言py","user_id":3107856775},"user_id":3107856775,"message_id":226595978,"font":0,"group_id":542926457,"message_type":"group"}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 14:33:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 14:33:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 14:33:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 14:33:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 14:33:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 14:33:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 14:33:25] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674887592,"self_id":3351593463,"sub_type":"normal","anonymous":null,"group_id":542926457,"message_type":"group","raw_message":"那就不改了","sender":{"age":0,"area":"","card":"","level":"","nickname":"洛瑜","role":"member","sex":"unknown","title":"","user_id":672545360},"message_id":-1802975952,"font":0,"message":"那就不改了","message_seq":18006,"user_id":672545360}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 14:33:27] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674887594,"self_id":3351593463,"sub_type":"normal","font":0,"message":"[CQ:face,id=277]","message_type":"group","sender":{"age":0,"area":"","card":"","level":"","nickname":"洛瑜","role":"member","sex":"unknown","title":"","user_id":672545360},"message_id":-477104730,"anonymous":null,"message_seq":18007,"raw_message":"[CQ:face,id=277]","user_id":672545360,"group_id":542926457}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 14:35:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 14:35:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 14:35:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 14:35:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 14:35:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 14:35:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 14:35:14] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674887701,"self_id":3351593463,"sub_type":"normal","anonymous":null,"message":"对了pr咋提交","message_type":"group","sender":{"age":0,"area":"","card":"","level":"","nickname":"jerry","role":"admin","sex":"unknown","title":"易语言py","user_id":3107856775},"user_id":3107856775,"font":0,"group_id":542926457,"message_seq":18008,"raw_message":"对了pr咋提交","message_id":1932585015}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 14:35:17] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674887704,"self_id":3351593463,"sub_type":"normal","user_id":3107856775,"font":0,"message":"[CQ:image,file=48da63f0dd18c3e070b643357c64ccc5.image,subType=7,url=https://gchat.qpic.cn/gchatpic_new/3107856775/542926457-2803491552-48DA63F0DD18C3E070B643357C64CCC5/0?term=3\u0026amp;is_origin=0]","message_seq":18009,"raw_message":"[CQ:image,file=48da63f0dd18c3e070b643357c64ccc5.image,subType=7,url=https://gchat.qpic.cn/gchatpic_new/3107856775/542926457-2803491552-48DA63F0DD18C3E070B643357C64CCC5/0?term=3\u0026amp;is_origin=0]","message_id":70768801,"anonymous":null,"group_id":542926457,"message_type":"group","sender":{"age":0,"area":"","card":"","level":"","nickname":"jerry","role":"admin","sex":"unknown","title":"易语言py","user_id":3107856775}}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 15:49:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 15:49:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 15:49:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 15:49:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 15:49:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 15:49:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 15:49:50] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"guild","time":1674892176,"self_id":3351593463,"sub_type":"channel","guild_id":"40345291637671734","channel_id":"1892244","message_id":"BACPVdQ4bEc2AAAAAAAc35QAAAAAAAAVdw==","user_id":"144115218678050588","message":"[CQ:at,qq=144115218680952989] /打卡 ","self_tiny_id":"144115218758343648","sender":{"nickname":"芈月.","tiny_id":"144115218678050588","user_id":144115218678050588}}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 15:49:51] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"guild","time":1674892178,"self_id":3351593463,"sub_type":"channel","user_id":"144115218680952989","message":"\n[CQ:at,qq=144115218678050588]   \n滴！打卡成功，赶快去#闲聊大厅活跃吧!\n也可以加入社区的新频道 【极域博客交流社区】  \n   \n📆连续打卡:1天，累计:111天，积分+5每日抽奖  \n恭喜你打卡成功，获得1次抽奖机会！  \n🔗\u003e\u003e\u003e立即抽奖  \n 红包抽奖*  \n兔年抽福袋！恭喜你获得1次抽奖机会，快来抽新春现金红包！\n🔗\u003e\u003e\u003e立即参与  \n   \n","self_tiny_id":"144115218758343648","sender":{"nickname":"小智管家","tiny_id":"144115218680952989","user_id":144115218680952989},"guild_id":"40345291637671734","channel_id":"1892244","message_id":"BACPVdQ4bEc2AAAAAAAc35QAAAAAAAAVeA=="}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 18:10:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 18:10:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 18:10:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 18:10:34] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"guild","time":1674900620,"self_id":3351593463,"sub_type":"channel","channel_id":"1678313","message_id":"BACPVdQ4bEc2AAAAAAAZm+kAAAAAAAAIDg==","user_id":"144115218185291865","message":"[CQ:at,qq=144115218709740871]偷偷地，别让人发现你与我相遇的喜悦。","self_tiny_id":"144115218758343648","sender":{"nickname":"频道助手","tiny_id":"144115218185291865","user_id":144115218185291865},"guild_id":"40345291637671734"}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 20:35:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 20:35:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 20:35:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 20:35:15] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674909301,"self_id":3351593463,"sub_type":"normal","raw_message":"为什么群聊发送链接时总是显示账号可能风控，但是私聊就能发[CQ:at,qq=2491267482] ","sender":{"age":0,"area":"","card":"","level":"","nickname":"万能的小黑","role":"member","sex":"unknown","title":"","user_id":2919465681},"message_id":1693471044,"anonymous":null,"font":0,"group_id":542926457,"message_type":"group","message":"为什么群聊发送链接时总是显示账号可能风控，但是私聊就能发[CQ:at,qq=2491267482] ","message_seq":18010,"user_id":2919465681}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 20:38:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 20:38:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 20:38:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 20:38:26] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674909493,"self_id":3351593463,"sub_type":"normal","sender":{"age":0,"area":"","card":"","level":"","nickname":"jerry","role":"admin","sex":"unknown","title":"易语言py","user_id":3107856775},"user_id":3107856775,"anonymous":null,"font":0,"group_id":542926457,"message":"[CQ:image,file=6c7dffc28d5c4e5c93b6cfefd70a8c05.image,subType=7,url=https://gchat.qpic.cn/gchatpic_new/3107856775/542926457-2166655758-6C7DFFC28D5C4E5C93B6CFEFD70A8C05/0?term=3\u0026amp;is_origin=0]","message_type":"group","raw_message":"[CQ:image,file=6c7dffc28d5c4e5c93b6cfefd70a8c05.image,subType=7,url=https://gchat.qpic.cn/gchatpic_new/3107856775/542926457-2166655758-6C7DFFC28D5C4E5C93B6CFEFD70A8C05/0?term=3\u0026amp;is_origin=0]","message_seq":18011,"message_id":334979538}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 20:38:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 20:38:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 20:38:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 20:38:44] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674909511,"self_id":3351593463,"sub_type":"normal","message_seq":18012,"sender":{"age":0,"area":"","card":"","level":"","nickname":"jerry","role":"admin","sex":"unknown","title":"易语言py","user_id":3107856775},"user_id":3107856775,"message_id":-1963052952,"anonymous":null,"message":"[CQ:reply,id=1693471044][CQ:at,qq=2919465681] [CQ:at,qq=2919465681] 你找点api过一下防红","message_type":"group","raw_message":"[CQ:reply,id=1693471044][CQ:at,qq=2919465681] [CQ:at,qq=2919465681] 你找点api过一下防红","font":0,"group_id":542926457}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 20:39:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 20:39:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 20:39:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 20:39:15] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674909542,"self_id":3351593463,"sub_type":"normal","message":"啊？没懂","message_type":"group","message_seq":18013,"raw_message":"啊？没懂","sender":{"age":0,"area":"","card":"","level":"","nickname":"万能的小黑","role":"member","sex":"unknown","title":"","user_id":2919465681},"message_id":-34012930,"group_id":542926457,"font":0,"user_id":2919465681,"anonymous":null}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 20:41:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 20:41:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 20:41:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 20:41:52] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674909699,"self_id":3351593463,"sub_type":"normal","user_id":3107856775,"font":0,"message_type":"group","message_seq":18014,"sender":{"age":0,"area":"","card":"","level":"","nickname":"jerry","role":"admin","sex":"unknown","title":"易语言py","user_id":3107856775},"message_id":1671271773,"anonymous":null,"group_id":542926457,"message":"就是链接报不安全","raw_message":"就是链接报不安全"}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 20:43:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 20:43:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 20:43:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 20:43:57] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674909824,"self_id":3351593463,"sub_type":"normal","user_id":2919465681,"anonymous":null,"font":0,"message_type":"group","sender":{"age":0,"area":"","card":"","level":"","nickname":"万能的小黑","role":"member","sex":"unknown","title":"","user_id":2919465681},"message_id":345679307,"group_id":542926457,"message":"但是私聊就可以发送","message_seq":18015,"raw_message":"但是私聊就可以发送"}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 20:43:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 20:44:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 20:44:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 20:44:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 20:44:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 20:44:08] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674909835,"self_id":3351593463,"sub_type":"normal","raw_message":"我发送的是CSDN链接","message_id":-1919683471,"anonymous":null,"font":0,"group_id":542926457,"message":"我发送的是CSDN链接","message_type":"group","message_seq":18016,"sender":{"age":0,"area":"","card":"","level":"","nickname":"万能的小黑","role":"member","sex":"unknown","title":"","user_id":2919465681},"user_id":2919465681}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 20:44:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 20:44:13] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674909840,"self_id":3351593463,"sub_type":"normal","font":0,"sender":{"age":0,"area":"","card":"","level":"","nickname":"jerry","role":"admin","sex":"unknown","title":"易语言py","user_id":3107856775},"user_id":3107856775,"message_id":-90913561,"message_seq":18017,"raw_message":"[CQ:image,file=6c7dffc28d5c4e5c93b6cfefd70a8c05.image,subType=7,url=https://gchat.qpic.cn/gchatpic_new/3107856775/542926457-2798878410-6C7DFFC28D5C4E5C93B6CFEFD70A8C05/0?term=3\u0026amp;is_origin=0]","anonymous":null,"group_id":542926457,"message":"[CQ:image,file=6c7dffc28d5c4e5c93b6cfefd70a8c05.image,subType=7,url=https://gchat.qpic.cn/gchatpic_new/3107856775/542926457-2798878410-6C7DFFC28D5C4E5C93B6CFEFD70A8C05/0?term=3\u0026amp;is_origin=0]","message_type":"group"}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 20:44:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 20:44:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 20:44:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 20:44:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 20:44:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 20:44:26] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674909852,"self_id":3351593463,"sub_type":"normal","anonymous":null,"message_type":"group","message_seq":18018,"raw_message":"[CQ:image,file=48da63f0dd18c3e070b643357c64ccc5.image,subType=7,url=https://gchat.qpic.cn/gchatpic_new/3107856775/542926457-2986472274-48DA63F0DD18C3E070B643357C64CCC5/0?term=3\u0026amp;is_origin=0]","sender":{"age":0,"area":"","card":"","level":"","nickname":"jerry","role":"admin","sex":"unknown","title":"易语言py","user_id":3107856775},"user_id":3107856775,"font":0,"group_id":542926457,"message":"[CQ:image,file=48da63f0dd18c3e070b643357c64ccc5.image,subType=7,url=https://gchat.qpic.cn/gchatpic_new/3107856775/542926457-2986472274-48DA63F0DD18C3E070B643357C64CCC5/0?term=3\u0026amp;is_origin=0]","message_id":1781258614}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 20:44:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 20:44:30] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674909857,"self_id":3351593463,"sub_type":"normal","group_id":542926457,"message_type":"group","sender":{"age":0,"area":"","card":"","level":"","nickname":"jerry","role":"admin","sex":"unknown","title":"易语言py","user_id":3107856775},"message_id":489482720,"anonymous":null,"font":0,"raw_message":"你换个号试试","user_id":3107856775,"message":"你换个号试试","message_seq":18019}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2023-01-28 20:45:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2023-01-28 20:45:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2023-01-28 20:45:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2023-01-28 20:45:23] [WARNING]: 上报 Event 数据 {"post_type":"message","message_type":"group","time":1674909910,"self_id":3351593463,"sub_type":"normal","anonymous":null,"font":0,"raw_message":"OK，等会换个号试试","sender":{"age":0,"area":"","card":"","level":"","nickname":"万能的小黑","role":"member","sex":"unknown","title":"","user_id":2919465681},"message_id":1339884167,"group_id":542926457,"message":"OK，等会换个号试试","message_type":"group","message_seq":18020,"user_id":2919465681}
 到 http://127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
